showcase/src/app/home/home.component.html

```diff
@@ -1,100 +1,114 @@
-<h1 class="mat-display-2" translate>Stark documentation</h1>
+<div class="about-background">
+	<h1 class="about-title" [innerHTML]= "'SHOWCASE.HOMEPAGE.MAIN_TITLE' | translate"></h1>
+</div>
+<h2 class="mat-body-2" [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DESC_BEFORE_DETAIL' | translate">
```

Comment:
I think you can place this content inside of a `<section class=".stark-section">`